x-pack/legacy/plugins/siem/public/containers/case/types.ts

```diff
@@ -16,6 +16,7 @@ export interface Comment {
 export interface Case {
   id: string;
   comments: Comment[];
+  commentIds: string[];
```

Comment:
Why we need the comment ids in a different array?  Aren't includes in comments array ?

Reply:
@XavierM had reasons

Reply:
@cnasikas, you will only get the comments through the API if you set the query parameter `includeComments` but we still need to know how many comments are attached to a case to show it in the table.

Reply:
Sorry to insist, but if this is the only use case why the API does not return the count of the comments? Do you think there will be another use case in the future that the ids will be needed?

Reply:
I agree with you but since we do not have aggregation on the saved object I did that to avoid query every comment for every case. Might not be the most robust way of doing it. However, it will be faster. When we have aggregation in saved_object, we will be able to use that to just get the count like you said